aten/src/ATen/TensorIterator.cpp

```diff
@@ -665,29 +666,7 @@ void TensorIteratorBase::serial_for_each(loop2d_t loop, Range range) const {
   if (range.size() == 0) {
     return;
   }
-  auto strides = get_strides();
-  while (strides.size() < 2U * ntensors()) {
-    strides.push_back(0);
-  }
-
-
-  auto base_ptrs = get_base_ptrs();
-  if (ndim() <= 1) {
-    if (range.begin > 0) {
-      auto ptrs = get_data_ptrs(base_ptrs, {range.begin});
```

Comment:
looks like you no longer need `get_data_ptrs` in TensorIterator.cpp since you have it in TensorIteratorInternal.h?